src/clarity-angular/layout/vertical-nav/_vertical-nav.clarity.scss

```diff
@@ -229,6 +264,18 @@ $vertical-nav-divider-margin: $clr_baselineRem_0_5;
             }
         }
 
+        .nav-group.active:not(.is-expanded) {
+            .nav-group-content {
+                background-color: $vertical-nav-hover-bg-color;
+            }
+        }
+
+        .nav-group-content {
+            .nav-link.active ~ .nav-group-trigger {
+                background-color: $vertical-nav-hover-bg-color;
```

Comment:
Not the nesting, the idea that the white background for a group is actually two white backgrounds tightly stuck together and relying on a `~` selector.

Feels like:

![387922](https://user-images.githubusercontent.com/4647197/29345490-276f9b3c-820d-11e7-8174-c41c06256870.jpg)